src/encoded/types/ingestion.py

```diff
@@ -207,24 +207,27 @@ def processing_context(self):
                               s3_encrypt_key_id=s3_encrypt_key_id) as fp:
             PRINT(json.dumps(resolution, indent=2), file=fp)
 
-    def process_standard_bundle_results(self, bundle_result):
+    def process_standard_bundle_results(self, bundle_result, s3_only=False):
```

Comment:
Without a doc string (sorry there isn't one already there to add to, I'll think about adding some that you could add to), it was not initially obvious to me what s3_only is doing here because I was hopping around. I did find your explanation elsewhere at some point. But just reading this on its own, it leaves a weird question of: why does this want to be suppressing results from visibility?

We might want to discuss with others what the right paradigm is to support a good UX paradigm. It could be that the result was the name of a file object. Like we could create another FileOther that is the result and that someone could then download if they wanted to see it. In that case, the result might contain a summary of how many of each category, though maybe that's in text as well (perhaps harder to parse so a program-readable summary as JSON might be good).

This may be a confusion about the use of this function. This is specifically so that you CAN report something to the user, so saying "here's something to report to the user but don't report  it to the user" seems odd. I think if the ingester is just supposed to do a thing and put it somewhere, it might be it should do it directly via lower level abstractions that are not the "result", etc. We could maybe discuss this.

Reply:
I added docstring comments to this function. The s3_only doesn't suppress all visibility, it just confines that visibility to a (submission.json) file in S3, and does not write it to the database; the results in question here are the detailed list of uuids processed, grouped by category (i.e. post, patch, skip, error), in JSON format. What *is* written to the database (via the note_additional_datum call in ingestion.processors.handle_ontology_update) is the *summary* (just the counts - in string/text format within a JSON object) and this is also presented to the user as summary output from submit-ontology.